spec/swagger_helper.rb

```diff
@@ -30,6 +30,280 @@
+                         "Art and design",
+                         "Biology",
+                         "Business Studies",
+                         "Chemistry",
+                         "Citizenship",
+                         "Classics",
+                         "Computing",
+                         "Dance",
+                         "Design And Technology",
+                         "Drama",
+                         "Economics",
+                         "Engineering",
+                         "English",
+                         "Food Technology",
+                         "French",
+                         "Geography",
+                         "German",
+                         "health_and_social_care",
+                         "history",
+                         "Humanities",
+                         "ICT",
+                         "Languages",
+                         "Law",
+                         "Mandarin",
+                         "Mathematics",
+                         "Media Studies",
+                         "Music",
+                         "Philosophy",
+                         "Physical Education",
+                         "Physics",
+                         "PSHE",
+                         "Psychology",
+                         "Religious Education",
+                         "Science",
+                         "Social Science",
+                         "Sociology",
+                         "Spanish",
+                         "Statistics"],
+                },
+              },
+            },
+          },
+          bad_request_error: {
+            type: "object",
+            properties: {
+              errors: {
+                type: "array",
+                items: {
+                  type: "object",
+                  properties: {
+                    error: { type: "string", example: "Bad request" },
+                    message: { type: "string", example: "Request body could not be read properly" },
+                  },
+                  required: %w[error message],
+                },
+              },
+            },
+          },
+          unauthorized_error: {
+            type: "object",
+            properties: {
+              errors: {
+                type: "array",
+                items: {
+                  type: "object",
+                  properties: {
+                    error: { type: "string", example: "Unauthorized" },
+                    message: { type: "string", example: "Invalid API key" },
+                  },
+                  required: %w[error message],
+                },
+              },
+            },
+          },
+          not_found_error: {
+            type: "object",
+            properties: {
+              errors: {
+                type: "array",
+                items: {
+                  type: "object",
+                  properties: {
+                    error: { type: "string", example: "Not found" },
```

Comment:
do we need both the 'error' and the 'message' in these schemas? It might be confusing to caller to know which one to look at.

Reply:
Good point. We have declared the specific HTTP error codes, so the "error" key is redundant, you're right.

I was leaving room for multiple errors being returned. But this is not the case (besides on validations) since they're mutually exclusive. 

I'm going to update these schemas/examples. Thanks for bringing this up!